tf2/src/cache.cpp

```diff
@@ -267,20 +267,7 @@ bool TimeCache::insertData(const TransformStorage& new_data, std::string* error_
       break;
     storage_it++;
   }
-  if (storage_it != storage_.end() && storage_it->stamp_ == new_data.stamp_)
-  {
-    if (error_str)
-    {
-      std::stringstream ss;
-      ss << "TF_REPEATED_DATA ignoring data with redundant timestamp";
-      *error_str = ss.str();
-    }
-    return false;
-  }
-  else
-  {
-    storage_.insert(storage_it, new_data);
-  }
+  storage_.insert(storage_it, new_data);
```

Comment:
Would it make sense to put a comment in here stating that repeated data is considered normal so that we don't get future PRs that change the behavior back?

Also, I haven't looked at the surrounding code too deeply, but if we find a transform with the same stamp in the `storage_`, should we do this?

```
if (storage_it != storage_.end() && storage_it->stamp_ == new_data.stamp_)
{
  *storage_it = new_data;
}
else
{
  storage_.insert(storage_it, new_data);
}
```

Apologies if I'm misinterpreting what's being done here and how `storage_` is used later.

Reply:
I'll check what pruneList do and look if I can just replace the old data.
I agree for the comment.